docs/protocol/extension_generate.md

```diff
@@ -102,9 +102,9 @@ return an error.
 
 #### Parameters
 
-The *$parameters* JSON describes zero or more “name”/”value” pairs,
+The `$parameters` JSON describes zero or more “name”/”value” pairs,
 where the “name” is the name of the parameter and the “value” is a
-$string, $number, or $boolean.
+`$string`, `$number`, or `$boolean`.
```

Comment:
Can confirm this works in the CI run: https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/pipelines/11503954
![image](https://github.com/triton-inference-server/server/assets/124622579/d29f28f8-e24b-45b9-a19a-c0b1c97c2b14)